src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filtredSex = people.filter(person => person.sex === 'm');
+
+  if (arguments.length === 2) {
```

Comment:
```suggestion
  if (century) {
```
It's better to check exactly what we need. In this case, we need to check if there is an argument 'century'.
What if the function will change someday and will be receiving 3 arguments instead of 2? Then your check won't work